docs/content/toolbar.md

```diff
@@ -74,3 +74,10 @@ Path to your SpatialOS Unreal GDK snapshot.
 
 * **Snapshot file name**</br>
 Name of your SpatialOS Unreal GDK snapshot file.
+
+* **Singleton classes**</br>
+Specifies which classes that should be generated as singleton classes during the [ICG process](./interop.md).
```

Comment:
Specifies **the** classes that ...

Reply:
Suggest simplifying the sentence.
Does the user have to add the class here? If so, make it clear that our user has to do something for the ICG to do something. Suggest:
```
Use this to specify the classes which the [ICG](./interop.md) should generate as singleton classes.

```

If the user uses this to just passively look at what the classes which must be singleton classes:

```
Lists the classes which the [ICG](./interop.md) generates as singleton classes.

```